src/panels/PanelInlineErrors/PanelInlineErrors.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from "@testing-library/react";
+
+import PanelInlineErrors from "./PanelInlineErrors";
+
+describe("PanelInlineErrors", () => {
```

Comment:
I think it would be good to also test that the `ScrollOnRender` component does not appear if `inlineErrors` is `null`, an empty array or only contains falsey items.